pyproject.toml

```diff
@@ -55,14 +50,16 @@ influxdb = "^5.3.1"
 # required for tests to pass right now. There are often issues when pinning to a branch
 # and calling poetry update. I have had to completely remove my venv and reinstall in order
 # to get the latest updates.
-alfalfa-client = "^0.4.0"
-autopep8 = "^1.4.4"
-pre-commit = "^2.20.0"
-pytest = "^6.2.5"
-pycodestyle = "^2.5.0"
-pytest-cov = "^3.0.0"
-pytest-timeout = "1.4.2"
-tox = "^3.21.4"
+# alfalfa-client = "^0.4.0"
+# alfalfa-client = {path = "../alfalfa-client"}
+alfalfa-client = { git = "https://github.com/nrel/alfalfa-client.git", branch = "upgrade-dependencies" }
```

Comment:
comment this and uncomment the version'd release above, once `alfalfa-client` is released.